src/processes.c

```diff
@@ -2029,6 +2029,8 @@ static int ps_read (void)
 	int stopped  = 0;
 	int paging   = 0;
 	int blocked  = 0;
+        int num_cmdline = 0;
```

Comment:
The formatting seems off here but in vi it looks fine. Unclear if it's a tab/ space thing or a Github UI bug. Also below in the first part of this review.

Reply:
PTAL